parsers/asm.c

```diff
@@ -208,9 +208,12 @@ static void makeAsmTag (
 													 kind_for_directive);
 				break;
 			case K_PSUEDO_MACRO_END:
-				macro_tag = getEntryInCorkQueue (*lastMacroCorkIndex);
-				macro_tag->extensionFields.endLine = getInputLineNumber ();
-				*lastMacroCorkIndex = CORK_NIL;
+				if (*lastMacroCorkIndex != CORK_NIL)
+				{
+					macro_tag = getEntryInCorkQueue (*lastMacroCorkIndex);
+					macro_tag->extensionFields.endLine = getInputLineNumber ();
+					*lastMacroCorkIndex = CORK_NIL;
+				}
```

Comment:
Yes, however, in that case I got a crash report. As far as my understanding, if lastMacroCorkIndex is non-zero,  macro_tag should have value. If this assumption is not satisfied, there is another bug I injected.